src/app/[locale]/(main)/imprint/page.tsx

```diff
@@ -0,0 +1,62 @@
+import '@/themes/theme.karlsruhe.scss';
+
+import { useTranslations } from 'next-intl';
+
+export default function Imprint() {
+  const t = useTranslations('Footer');
+  return (
+    <div className="container mt-4">
+      <h1 className="custom-heading">{t('imprint')}</h1>
```

Comment:
I would remove the `custom-heading` class. We should keep the headings consistent throughout the application.